### ISSUE TITLE
Package 2.46 for Debian (thus Ubuntu)

### ISSUE THREAD

**Issue by p-bo** [pixelb/fslint#138]
V 2.44, currently packaged in Debian and Ubuntu, has lot of deficiencies according:

https://raw.githubusercontent.com/pixelb/fslint/master/doc/NEWS

So would it be possible to resolve this packaging request (you are maintainer of package in Debian)? :

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=858772

Next LTS version of Ubuntu (18.04) is nearing and it would be unfortunate to have there 4 years (in time of OS release) old buggy release...

Many thanks :-)

**Comment by clickwir** [comment]
I agree, I'd love to see a newer version included. I was running 2.44 a few minutes ago and it just didn't seem to be deleting duplicates if there were more than a few selected. Upgraded to 2.47 and works fine. 

Upgraded using the following instructions:

git clone https://github.com/pixelb/fslint.git fslint-2.47
cd fslint-2.47
dpkg-buildpackage -I.git -rfakeroot -tc -us -uc
sudo dpkg -i ../fslint_2.47-1_all.deb

**Comment by Baltix** [comment]
fslint will be removed from Debian next month, because it uses 7 years old unmaintained pygtk libraries.
pygtk is unmaintained upstream. It has not had a release since GNOME 3 was released in 2011 !
The way forward is to port your app to use GObject Introspection bindings.

For more information on GObject Introspection see [1] and [2].

Please try to do this before the Debian "Buster" release as we're going to try to remove pygtk this cycle.
[1] https://wiki.gnome.org/Projects/GObjectIntrospection
[2] https://wiki.gnome.org/Projects/PyGObject

See https://bugs.debian.org/885280 for more info

**Comment by Valeryan24** [comment]
Now, FSLint has been removed from Debian testing, unstable and Ubuntu 20.04 due to this reason.

I hope there will be an update or fork, because it was the best program to manage duplicates. GUI of RMLint is far less easier and complete, and CLI like fdupes doesn't allow to scan dozen of (sub)-folders + suppress in just one click all duplicates from a same folder : you have to choose for each set of duplicates which ones you keep or delete...
